azure-pipelines-dash.yml

```diff
@@ -0,0 +1,75 @@
+# C/C++ with GCC
+# Build your C/C++ project with GCC using make.
+# Add steps that publish test results, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/apps/c-cpp/gcc
+
+pr:
+- dash
+
+trigger:
+  batch: true
+  branches:
+    include:
+    - dash
+
+# this part need to be set in UI
+schedules:
+- cron: "0 0 * * 6"
+  displayName: Weekly build
+  branches:
+    include:
+    - dash
+  always: true
+
+variables:
+  - name: BUILD_BRANCH
+    ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
+      value: $(System.PullRequest.TargetBranch)
+    ${{ else }}:
+      value: $(Build.SourceBranchName)
+
+stages:
+- stage: Build
+  jobs:
+  - template: .azure-pipelines/build-template.yml
+    parameters:
+      arch: amd64
+      sonic_slave: sonic-slave-bullseye
+      swss_common_artifact_name: sonic-swss-common
+      swss_common_branch: master
+      artifact_name: sonic-swss
+      archive_pytests: true
+      archive_gcov: true
+      sairedis_artifact_name: sonic-sairedis
+      sairedis_artifact_branch: master
+      common_lib_artifact_name: common-lib
+      common_lib_artifact_branch: master
+
+
+- stage: BuildDocker
+  dependsOn: Build
+  condition: succeeded('Build')
+  jobs:
+  - template: .azure-pipelines/build-docker-sonic-vs-template.yml
+    parameters:
+      buildimage_artifact_branch: master
```

Comment:
When your PR is merged, and checked out 202311 branch. You are still using master branch's dependency.
In future, maybe 202411, your pipeline will fail.

Reply:
I feel this Azp file will be removed after this PR is merged.  It's just for a specific pipeline of DASH.